### DISCUSSION TITLE
AirPlay through BBN

### DISCUSSION THREAD

**Discussion by wsturner88** [bareboat-necessities/lysmarine_gen#415] [General]
I noticed I can now use airplay through BBN.  This is perfect for my boats sound system.    I have the hardware setup, out audio jack and into AUX on my audio system.   It's not working, but I am guessing I need to route the output through the audio jack and not HDMI.   Any recommendations for making this change?  
BTW - what is the app called that is offering this service?

**Comment by mgrouch** [comment]
It’s shairport-sync. 

https://github.com/juhovh/shairplay

I’d guess somewhere in its config you need to switch to aux. bookworm is using pipewire.

**Reply by wsturner88** [reply]
It looks like the audio output is controled when you build the program.   Not quite sure what this means.
Needs to have the -JACK extension
--with-jack
To build Shairport Sync for AirPlay 2, include the --with-airplay-2 option in the ./configure ...

Is this something you can do in your next build?

**Reply by mgrouch** [reply]
You do not need that. It’s already configured to output to pulseaudio (pa). And PA is defaulted to HDMI. You just need to switch PulseAudio to output to aux
